src/App.js

```diff
@@ -12,13 +12,80 @@ const goodsFromServer = [
   'Honey',
   'Jam',
   'Garlic',
-];
-
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+].map((good, index) => ({
+  product: good,
+  id: index + 1,
+}));
+
+let isResetted = false;
+
+class App extends React.Component {
+  state = {
+    selectedGood: ['Jam'],
+  }
```

Comment:
Цю змінну можна винести в state і використовівати звідти
```suggestion
class App extends React.Component {
  state = {
    selectedGood: ['Jam'],
    isResetted: false,
  }
```